docs/openapi.yml

```diff
@@ -340,6 +340,12 @@ paths:
       operationId: createCohort
       security:
         - bearerAuth: []
+      requestBody:
+        description: success
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/createCohort'
```

Comment:
The casing isn't right here, should be pascal case because the schema is called `CreateCohort`